### PR TITLE
Mx 13962 consistent tokens values length checks

### PR DESCRIPTION
**Pull request by bogdan-rosianu**
## Reasoning behind the pull request
- changes in vm-common should be reflected inside mx-chain-go
  
## Proposed changes
- integrate new vm-common + update interface / config
- should be reviewed with: https://github.com/multiversx/mx-chain-vm-common-go/pull/206

## Testing procedure
- regular all-in

## Pre-requisites

Based on the [Contributing Guidelines](https://github.com/multiversx/mx-chain-go/blob/master/.github/CONTRIBUTING.md#branches-management) the PR author and the reviewers must check the following requirements are met:
- was the PR targeted to the correct branch?
- if this is a larger feature that probably needs more than one PR, is there a `feat` branch created?
- if this is a `feat` branch merging, do all satellite projects have a proper tag inside `go.mod`?

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -19,7 +19,7 @@ require (
 	github.com/multiversx/mx-chain-logger-go v1.0.11
 	github.com/multiversx/mx-chain-p2p-go v1.0.16
 	github.com/multiversx/mx-chain-storage-go v1.0.8
-	github.com/multiversx/mx-chain-vm-common-go v1.4.1
+	github.com/multiversx/mx-chain-vm-common-go v1.4.2-0.20230515133310-3417cce7427e
```
Comment (sasurobert) [inline go.mod:22]: proper release

File: cmd/node/config/enableEpochs.toml [hunk 2]
```diff
@@ -250,13 +250,16 @@
 
     # BLSMultiSignerEnableEpoch represents the activation epoch for different types of BLS multi-signers
     BLSMultiSignerEnableEpoch = [
-        { EnableEpoch = 0, Type = "no-KOSK"},
-        { EnableEpoch = 1, Type = "KOSK"}
+        { EnableEpoch = 0, Type = "no-KOSK" },
+        { EnableEpoch = 1, Type = "KOSK" }
     ]
 
     # SetGuardianEnableEpoch represents the epoch when the guard account feature is enabled in the protocol
```
Comment (iulianpascalau) [inline cmd/node/config/enableEpochs.toml:257]: can we move L257 - L262 after the `BLSMultiSignerEnableEpoch` block?

Comment (bogdan-rosianu) [reply cmd/node/config/enableEpochs.toml:254]: moved

File: config/tomlConfig_test.go [hunk 3]
```diff
@@ -799,6 +802,7 @@ func TestEnableEpochConfig(t *testing.T) {
 			SetGuardianEnableEpoch:                       64,
 			MultiClaimOnDelegationEnableEpoch:            65,
 			KeepExecOrderOnCreatedSCRsEnableEpoch:        64,
```
Comment (iulianpascalau) [inline config/tomlConfig_test.go:804]: inconsistencies after merge

**Review by gabi-vuls — DISMISSED** [review]
System test passed

**Review by sasurobert — APPROVED** [review]

**Review by iulianpascalau — APPROVED** [review]

**Review by gabi-vuls — APPROVED** [review]

**Pull request merged**